modules/nodes-containers-volumes-adding.adoc

```diff
@@ -37,11 +37,11 @@ character.
 |`'*'`
 
 |`-m, --mount-path`
-|Mount path inside the selected containers.
+|Mount path inside the selected containers. Do not mount to the container root, `/`, or any path that is the same in the host and the container, as it can corrupt your host system. It is safe to mount the host by using `/host`.
```

Comment:
`as it can corrupt your host system if the container is sufficiently privileged`
I like this better because it doesn't imply there are simple container breakouts from malicious users, but rather problems with users mistakenly specifying the paths